### PR TITLE
FIX: remove the LNURLp metadata validation

### PR DESCRIPTION
**Pull request by Overtorment**
https://twitter.com/callebtc/status/1752643575274152281

https://github.com/lnurl/luds/pull/234

### CODE REVIEW

**Review by marcosrdz — APPROVED** [review]

**Review by limpbrains — APPROVED** [review]

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/pdsxhitoqrk6x5bor73yilaa4u?device=pixel4

[download apk](https://lambda-download-android-build.herokuapp.com/download/327)

File: class/lnurl.js [hunk 1]
```diff
@@ -146,7 +146,7 @@ export default class Lnurl {
     const decoded = this.decodeInvoice(this._lnurlPayServiceBolt11Payload.pr);
     const metadataHash = createHash('sha256').update(this._lnurlPayServicePayload.metadata).digest('hex');
     if (metadataHash !== decoded.description_hash) {
-      throw new Error(`Invoice description_hash doesn't match metadata.`);
+      console.log(`Invoice description_hash doesn't match metadata.`);
     }
```
Comment (callebtc) [inline class/lnurl.js:150]: Why not just remove all of this?

Comment (Overtorment) [reply class/lnurl.js:150]: 🤔

**Review by jaonoctus — COMMENTED** [review]
post-merge tACK

**Pull request merged**